app/controllers/staff/level_b_activity_uploads_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Staff::LevelBActivityUploadsController < Staff::BaseController
+  include Secured
+
+  def new
+    @organisation ||= Organisation.find(params[:organisation_id])
+    authorize @organisation, :show_bulk_upload?
```

Comment:
Do we foresee having different levels of bulk upload authorisations? I.e. some users can only `show_bulk_upload`, whereas others can `create_bulk_upload`?

Reply:
We've now renamed this to just `bulk_upload?` as there isn't a difference, and "showing" isn't really the important verb here.